### DISCUSSION TITLE
Ranger方式认证，SWITCH CATALOG和use database都没问题，但是查询数据表有权限错误

### DISCUSSION THREAD

**Discussion by colinguozizhong** [apache/doris#20454] [A - General / Q&A]
ERROR 1105 (HY000): errCode = 2, detailMessage = Unexpected exception: failed to get table tmp_txzl_as2 in db guozz from hms client. reason: org.apache.hadoop.hive.metastore.api.MetaException: java.security.AccessControlException: Permission denied: user=root, access=EXECUTE, inode="/warehouse/tablespace/managed/hive":hive:hadoop:drwx------
        at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.check(FSPermissionChecker.java:399)
        at org.apache.hadoop.hdfs.server.namenode.FSPermissionChecker.checkTraverse(FSPermission

但是在catalog中配置是有'hadoop.username' = 'hive',参数，为啥访问还是root账户呢？

**Comment by yagagagaga** [comment]
You can add `HADOOP_USER_NAME =hive` to your fe.conf and be.conf and try again.
